packages/chaire-lib-backend/src/utils/processManagers/OSRMProcessManager.ts

```diff
@@ -61,28 +63,34 @@ const availableRoutingModes = async function (): Promise<RoutingMode[]> {
 
 function errorConfiguringMode(mode: RoutingMode, message: string) {
     console.log('Error configuring OSRM mode ' + mode + ' ' + message);
-    const updatedProperties = {};
-    updatedProperties['modes'] = {};
-    updatedProperties['modes'][mode] = { enabled: false };
-    Preferences.updateServerPrefs({ osrmRouting: updatedProperties });
+    setProjectConfiguration({
+        routing: {
+            [mode]: {
+                defaultEngine: 'osrmRouting',
+                engines: {
+                    osrmRouting: {enabled: false}
```

Comment:
Also, please run `yarn format` before committing as it will change the formatting

Reply:
I ran the command but it didn't change this part specifically